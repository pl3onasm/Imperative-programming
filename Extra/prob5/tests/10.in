1510656521 1510795652
