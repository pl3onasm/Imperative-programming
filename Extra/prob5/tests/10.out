1510697005: 1011010000010110110100000101101
1510729773: 1011010000010111110100000101101
1510742061: 1011010000011000001100000101101
1510774829: 1011010000011001001100000101101
